src/code/fv-components/fv-gamete-image.js

```diff
@@ -28,18 +28,19 @@ const FVGameteImageView = ({chromosomes=[], id, className, style, displayStyle})
     chromosomeImages.push(
       <ChromosomeImageView
         key={i}
-        empty={!chromosome}
-        width={6 * scale}
-        height={(chromosome && chromosome.side === "y" ? 15 : 20) * scale}
-        split={8 * scale}
+        empty={false}
```

Comment:
These are the old SVG chromosomes, so we're not going to be using them for very long in any case. Empty means show the outline but don't fill the interior. Previously, we distinguished selection by having a fillColor versus being empty. To look more like the FableVision art in the short term, I simply have it always fill (i.e. never empty) with either the dark or light blue chromosome color. It will all go away when we start using the FableVision images, but this seemed like a simple quick-fix to improve the look and could be good enough for the classroom test.